### ISSUE TITLE
vjunosswitch netconf over ssh issue

### ISSUE THREAD

**Issue by HeikkiLavaste** [hellt/vrnetlab#186]
Hi, 

I have the followng issue
I've created a vjunos-switch image using image vJunos-switch-23.1R1.8.qcow2. 
I'll start the container using containerlabs. SSH works but netconf over ssh does not. 
What's missing is the line "set system services netconf ssh"
Looking at the files that are within the docker container, that should ge applied during startup? 
I don't see any errors printed to docker logs during startup. 

Thanks, 
Heikki

**Comment by hellt** [comment]
if you examine the full boot log (`docker logs <node name>`) don't you see this command applied at run time?

**Comment by HeikkiLavaste** [comment]
Hi, 

I can see the following in the logs: 

2024-04-10 14:24:41,477: launch     TRACE    Startup config file /config/startup-config.cfg is not found
'juniper.conf' -> '/var/tmp/tmp.4XvpJGmoBI/config/juniper.conf'
Formatting 'config.img', fmt=qcow2 size=1048576 cluster_size=65536 lazy_refcounts=off refcount_bits=16
losetup: config.img: Warning: file does not fit into a 512-byte sector; the end of the file will be ignored.
Copying file(s) to config disk config.img
./config/
./config/juniper.conf
removed '/var/tmp/tmp.4XvpJGmoBI/config/juniper.conf'
removed directory '/var/tmp/tmp.4XvpJGmoBI/config'
Config disk config.img created

If I connect to the container, the juniper.conf does include the config: 

root@vjunos:/# cat juniper.conf 
system {
    host-name vjunos;
    root-authentication {
        plain-text-password-value "admin@123";
    }
    login {
        user admin {
            class super-user;
            authentication {
                plain-text-password-value "admin@123";
            }
        }
    }
    services {
        ssh {
            root-login allow;
        }
        netconf {
            ssh;
        }
    }
}
interfaces {
    fxp0 {
        unit 0 {
            family inet {
                address 10.0.0.15/24;
            }
        }
    }
}

The other odd thing is that the admin password, when logging in, is not admin@123 but admin1. 
I removed the container image from local and created it again, thinking it was something I committed to the container after it was created.  The password is still admin1. 

Thanks, 
Heikki
